clusters/build-clusters/01_cluster/sshd-bastion/ppc64le-libvirt/coredns_config.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+data:
+  Corefile: |
+      .:5353 {
+            errors
+            log
+            health
+            rewrite name suffix libvirt-ppc64le-0-0 sshd-0-0.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-0-1 sshd-0-1.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-0-2 sshd-0-2.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-0-3 sshd-0-3.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-1-0 sshd-1-0.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-1-1 sshd-1-1.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-1-2 sshd-1-2.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-1-3 sshd-1-3.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-2-0 sshd-2-0.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-2-1 sshd-2-1.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-2-2 sshd-2-2.default.svc.cluster.local
+            rewrite name suffix libvirt-ppc64le-2-3 sshd-2-3.default.svc.cluster.local
+            kubernetes cluster.local 10.0.0.0/24
```

Comment:
we may want to try testing without this line, but I'm fine with removing it later if we know this works.

Reply:
Updated 👍🏻